android/app/src/main/jni/main.cpp

```diff
@@ -0,0 +1,96 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "string_script.inc"
+
+#include <android/log.h>
+#include <android_native_app_glue.h>
+#include <cassert>
+#include <vector>
+
+extern "C" {
```

Comment:
For what it’s worth, I changed the vkrunner headers so that they have the `extern "C"` in it themselves so they can just be directly included from C++ now.

Reply:
Thanks!